test/CreatorToken.t.sol

```diff
@@ -484,3 +497,51 @@ contract Pausing is CreatorTokenTest {
     vm.stopPrank();
   }
 }
+
+contract bondingCurveTests is CreatorTokenTest {
```

Comment:
Nit: why a lowercase "b"?

Also, I would name this something a bit more clear. We are not testing the bonding curve (the production implementation of which will have its own unit tests). Rather, we are testing that the Creator Token contract properly *uses* the bonding curve to determine its pricing. So maybe something like `CreatorTokenFollowsBondingCurveContract` or something similar.

Reply:
Updated this in the same commit as rest of the naming updates: https://github.com/showtime-xyz/creator-tokens/commit/89f1666530442849e5fcc6a35261447ee2cf1cbf